packages/odyssey-react/src/components/Heading/Heading.module.scss

```diff
@@ -15,6 +15,7 @@
   margin-block-end: $spacing-xs-em;
   margin-inline: 0;
   color: $text-heading;
+  font-family: $body-font-family;
```

Comment:
?? whats this change about for `Heading` ?

Reply:
When the global styles are removed the font-family definition for h tags go with it. Adding it here ensures that the correct `font-family` is used.

It seemed correct to use it here because this is a composable component like text.

Reply:
Got it thank you